content/spin/rdbms-storage.md

```diff
@@ -32,6 +32,12 @@ The exact detail of calling these operations from your application depends on yo
 
 {{ startTab "Rust"}}
 
+**SQLite**
```

Comment:
The top of the page already links out to SQLite - remove this so it can stick to the API it's documenting.